duckduckgo/iOS
# Add Advanced Settings screen with Autoplay Media option

Task/Issue URL: https://app.asana.com/0/414235014887631/1153727682934211/f
Tech Design URL:
CC:

**Description**:

This PR adds a new Advanced Settings screen, featuring an Autoplay Media option.

**Steps to test this PR**:
1. Run the app without changing any settings and check that media does not autoplay, you can use [W3Schools](https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_video_autoplay) to test this
1. Change the setting in Settings -> Advanced Settings -> Autoplay Media and go back to the test page, verify that media autoplays
1. Kill the app and relaunch it, verify that the setting has the same value that you set it before closing the app



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
